index.js

```diff
@@ -17,6 +17,9 @@ module.exports = function({ bucket, manifestKey, healthCheckerUA, sentryDSN, fas
 
   fastbootConfig = {...FASTBOOT_DEFAULTS, ...fastbootConfig};
 
+  env = process.env.ENV || 'dev'
```

Comment:
I know this is annoying, but `ENV` is kind of generic for this use and we haven't really landed on a convention for using generic envvars yet. I'd be ok with using `ENV` as the name for this value, but only if we're sure that setting it to `prod` or `demo` won't mess with other parts of the build or the process itself.

Can you verify if `ENV` is used elsewhere? Or else name space the envvar that you're using for name spacing 😬 

also please declare new variables in scope by using `let` or `const`. probably `const` in this case.

Reply:
it's not used by anything else in the repo. However, what we could do is modify the method signature to put the burden of setting that variable entirely on the project that uses this library if you're more comfortable with that.